### PR TITLE
created registration form

### PR DESCRIPTION
**Pull request by parpinelopi**
Hello team, I have adjusted the registration form for our needs. Please review and get back to me.

### CODE REVIEW

**Review by elmenqui — APPROVED** [review]
LGTM

**Review by MeriM-dev — APPROVED** [review]
LGTM!

File: frontend/src/components/auth/RegisterForm.js [hunk 1]
```diff
@@ -2,13 +2,16 @@ import React, {useState} from "react";
 
 function RegisterForm({onSubmit}) {
     const [name, setName] = useState("");
+    const [id, setId]=useState("");
+    const [address, setAddress]=useState("");
+    const [telephone, setTelephone]=use
```
Comment (MeriM-dev) [inline frontend/src/components/auth/RegisterForm.js]: should this be `useState("");`?

File: frontend/src/components/auth/RegisterForm.js [hunk 2]
```diff
@@ -20,6 +23,26 @@ function RegisterForm({onSubmit}) {
...
+                            onChange={ e => setId(e.target.value) }
+                            placeholder="Id - YYYYMMDD-XXXX"/>
+                    </div>
+
+                    <div className="form-group">
+                        <label>Address:</label>
+                        <input 
+                            type="text"
+                            className="form-control"
+                            value={name}
```
Comment (MeriM-dev) [inline frontend/src/components/auth/RegisterForm.js]: This should be `value={address}`

File: frontend/src/components/auth/RegisterForm.js [hunk 3]
```diff
@@ -30,6 +53,16 @@ function RegisterForm({onSubmit}) {
                             placeholder="Email"/>
                     </div>
 
+                    <div className="form-group">
+                        <label>Telephone:</label>
+                        <input
+                            type="text"
+                            value={name}
```
Comment (MeriM-dev) [inline frontend/src/components/auth/RegisterForm.js]: This should be `value={telephone}`

**Review by MeriM-dev — CHANGES_REQUESTED** [review]

**Review by MeriM-dev — APPROVED** [review]

**Pull request merged**